week-6/Homework/mandatory/2-quotegenerator/quotes.js

```diff
@@ -1,3 +1,18 @@
+  setTimeout(()=>{
```

Comment:
And how can we switch off the auto mode? I think we need an extra button for it.